api/src/main/java/marquez/common/Utils.java

```diff
@@ -313,6 +316,21 @@ public static Version newDatasetVersionFor(
     return newDatasetVersionFor(data);
   }
 
+  /**
+   * Verifies if a job is a streaming job.
+   *
+   * @param job
+   * @return
+   */
+  public static boolean isStreamingJob(Job job) {
```

Comment:
_Minor_: Actually we can go one step further and define this method within the [`LineageEvent.Job`](https://github.com/MarquezProject/marquez/blob/main/api/src/main/java/marquez/service/models/LineageEvent.java#L204) class:

```
job.isStreaming()
```